UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/DartConfiguration.tcl
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 1
      Start  1: incorrect_solution_tests

1: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/incorrect_solution_tests
1: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
1: Test timeout computed to be: 5
1: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
1: [==========] Running 12 tests from 1 test suite.
1: [----------] Global test environment set-up.
1: [----------] 12 tests from LRUCacheTest
1: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
1: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
1: [ RUN      ] LRUCacheTest.CapacityAndSize
1: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
1: [ RUN      ] LRUCacheTest.BasicPutGet
1: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
1: [ RUN      ] LRUCacheTest.Eviction
1: [       OK ] LRUCacheTest.Eviction (0 ms)
1: [ RUN      ] LRUCacheTest.UsageOrderCheck
1: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
1: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
1: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (52 ms)
1: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
1: [       OK ] LRUCacheTest.ExpirationTime_BasicExpiry (64 ms)
1: [ RUN      ] LRUCacheTest.ExpirationTime_ResetOnAccess
1: [       OK ] LRUCacheTest.ExpirationTime_ResetOnAccess (217 ms)
1: [ RUN      ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
1: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp:151: Failure
1: Expected: switch (0) case 0: default: if (const ::testing::AssertionResult gtest_ar = (::testing::internal::EqHelper::Compare("cache.get(\"First\")", "111", cache.get("First"), 111))) ; else ::testing::internal::AssertHelper(::testing::TestPartResult::kNonFatalFailure, "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp", 151, gtest_ar.failure_message()) = ::testing::Message() doesn't throw an exception.
1:   Actual: it throws std::runtime_error with description "Key not found in cache".
1: 
1: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate (65 ms)
1: [ RUN      ] LRUCacheTest.PutAll_Basic
1: [       OK ] LRUCacheTest.PutAll_Basic (0 ms)
1: [ RUN      ] LRUCacheTest.PutAll_CapacityOverflow
1: [       OK ] LRUCacheTest.PutAll_CapacityOverflow (0 ms)
1: [ RUN      ] LRUCacheTest.PutAll_WithExpirationTime
1: [       OK ] LRUCacheTest.PutAll_WithExpirationTime (62 ms)
1: [----------] 12 tests from LRUCacheTest (462 ms total)
1: 
1: [----------] Global test environment tear-down
1: [==========] 12 tests from 1 test suite ran. (462 ms total)
1: [  PASSED  ] 11 tests.
1: [  FAILED  ] 1 test, listed below:
1: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
1: 
1:  1 FAILED TEST
 1/11 Test  #1: incorrect_solution_tests .........***Failed    0.78 sec
test 2
      Start  2: model1_tests

2: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model1_tests
2: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
2: Test timeout computed to be: 5
2: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
2: [==========] Running 12 tests from 1 test suite.
2: [----------] Global test environment set-up.
2: [----------] 12 tests from LRUCacheTest
2: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
2: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
2: [ RUN      ] LRUCacheTest.CapacityAndSize
2: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
2: [ RUN      ] LRUCacheTest.BasicPutGet
2: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
2: [ RUN      ] LRUCacheTest.Eviction
2: [       OK ] LRUCacheTest.Eviction (0 ms)
2: [ RUN      ] LRUCacheTest.UsageOrderCheck
2: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
2: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
2: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (55 ms)
2: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
2: [       OK ] LRUCacheTest.ExpirationTime_BasicExpiry (65 ms)
2: [ RUN      ] LRUCacheTest.ExpirationTime_ResetOnAccess
2: [       OK ] LRUCacheTest.ExpirationTime_ResetOnAccess (215 ms)
2: [ RUN      ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp:151: Failure
2: Expected: switch (0) case 0: default: if (const ::testing::AssertionResult gtest_ar = (::testing::internal::EqHelper::Compare("cache.get(\"First\")", "111", cache.get("First"), 111))) ; else ::testing::internal::AssertHelper(::testing::TestPartResult::kNonFatalFailure, "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp", 151, gtest_ar.failure_message()) = ::testing::Message() doesn't throw an exception.
2:   Actual: it throws std::runtime_error with description "Key not found in cache".
2: 
2: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate (65 ms)
2: [ RUN      ] LRUCacheTest.PutAll_Basic
2: [       OK ] LRUCacheTest.PutAll_Basic (0 ms)
2: [ RUN      ] LRUCacheTest.PutAll_CapacityOverflow
2: [       OK ] LRUCacheTest.PutAll_CapacityOverflow (0 ms)
2: [ RUN      ] LRUCacheTest.PutAll_WithExpirationTime
2: [       OK ] LRUCacheTest.PutAll_WithExpirationTime (63 ms)
2: [----------] 12 tests from LRUCacheTest (466 ms total)
2: 
2: [----------] Global test environment tear-down
2: [==========] 12 tests from 1 test suite ran. (466 ms total)
2: [  PASSED  ] 11 tests.
2: [  FAILED  ] 1 test, listed below:
2: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
2: 
2:  1 FAILED TEST
 2/11 Test  #2: model1_tests .....................***Failed    0.81 sec
test 3
      Start  3: model10_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/buUnable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
ild/Development/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model10_tests

3: Test command: 
3: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
 3/11 Test  #3: model10_tests ....................***Not Run   0.00 sec
test 4
      Start  4: model2_tests

4: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model2_tests
4: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
4: Test timeout computed to be: 5
4: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
4: [==========] Running 12 tests from 1 test suite.
4: [----------] Global test environment set-up.
4: [----------] 12 tests from LRUCacheTest
4: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
4: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
4: [ RUN      ] LRUCacheTest.CapacityAndSize
4: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
4: [ RUN      ] LRUCacheTest.BasicPutGet
4: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
4: [ RUN      ] LRUCacheTest.Eviction
4: [       OK ] LRUCacheTest.Eviction (0 ms)
4: [ RUN      ] LRUCacheTest.UsageOrderCheck
4: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
4: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
4: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (55 ms)
4: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
 4/11 Test  #4: model2_tests .....................***Exception: SegFault  0.49 sec
test 5
      Start  5: model3_tests

5: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model3_tests
5: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
5: Test timeout computed to be: 5
5: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
5: [==========] Running 12 tests from 1 test suite.
5: [----------] Global test environment set-up.
5: [----------] 12 tests from LRUCacheTest
5: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
5: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
5: [ RUN      ] LRUCacheTest.CapacityAndSize
5: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
5: [ RUN      ] LRUCacheTest.BasicPutGet
5: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
5: [ RUN      ] LRUCacheTest.Eviction
5: [       OK ] LRUCacheTest.Eviction (0 ms)
5: [ RUN      ] LRUCacheTest.UsageOrderCheck
5: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
5: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
5: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (54 ms)
5: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
5: [       OK ] LRUCacheTest.ExpirationTime_BasicExpiry (65 ms)
5: [ RUN      ] LRUCacheTest.ExpirationTime_ResetOnAccess
5: [       OK ] LRUCacheTest.ExpirationTime_ResetOnAccess (211 ms)
5: [ RUN      ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp:151: Failure
5: Expected: switch (0) case 0: default: if (const ::testing::AssertionResult gtest_ar = (::testing::internal::EqHelper::Compare("cache.get(\"First\")", "111", cache.get("First"), 111))) ; else ::testing::internal::AssertHelper(::testing::TestPartResult::kNonFatalFailure, "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp", 151, gtest_ar.failure_message()) = ::testing::Message() doesn't throw an exception.
5:   Actual: it throws std::runtime_error with description "Key has expired".
5: 
5: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate (65 ms)
5: [ RUN      ] LRUCacheTest.PutAll_Basic
5: [       OK ] LRUCacheTest.PutAll_Basic (0 ms)
5: [ RUN      ] LRUCacheTest.PutAll_CapUnable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
acityOverflow
5: [       OK ] LRUCacheTest.PutAll_CapacityOverflow (0 ms)
5: [ RUN      ] LRUCacheTest.PutAll_WithExpirationTime
5: [       OK ] LRUCacheTest.PutAll_WithExpirationTime (65 ms)
5: [----------] 12 tests from LRUCacheTest (463 ms total)
5: 
5: [----------] Global test environment tear-down
5: [==========] 12 tests from 1 test suite ran. (463 ms total)
5: [  PASSED  ] 11 tests.
5: [  FAILED  ] 1 test, listed below:
5: [  FAILED  ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
5: 
5:  1 FAILED TEST
 5/11 Test  #5: model3_tests .....................***Failed    0.79 sec
test 6
      Start  6: model4_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests

6: Test command: 
6: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
 6/11 Test  #6: model4_tests .....................***Not Run   0.00 sec
test 7
      Start  7: model5_tests

7: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model5_tests
7: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
7: Test timeout computed to be: 5
7: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
7: [==========] Running 12 tests from 1 test suite.
7: [----------] Global test environment set-up.
7: [----------] 12 tests from LRUCacheTest
7: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
7: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
7: [ RUN      ] LRUCacheTest.CapacityAndSize
7: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
7: [ RUN      ] LRUCacheTest.BasicPutGet
7: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
7: [ RUN      ] LRUCacheTest.Eviction
7: [       OK ] LRUCacheTest.Eviction (0 ms)
7: [ RUN      ] LRUCacheTest.UsageOrderCheck
7: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
7: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
7: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (55 ms)
7: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
 7/11 Test  #7: model5_tests .....................***Exception: SegFault  0.45 sec
test 8
      Start  8: model6_tests

8: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model6_tests
8: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tUnable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
asks/621208/build
8: Test timeout computed to be: 5
8: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/_deps/googletest-src/googletest/src/gtest_main.cc
8: [==========] Running 12 tests from 1 test suite.
8: [----------] Global test environment set-up.
8: [----------] 12 tests from LRUCacheTest
8: [ RUN      ] LRUCacheTest.GetFromEmptyCacheShouldThrow
8: [       OK ] LRUCacheTest.GetFromEmptyCacheShouldThrow (0 ms)
8: [ RUN      ] LRUCacheTest.CapacityAndSize
8: [       OK ] LRUCacheTest.CapacityAndSize (0 ms)
8: [ RUN      ] LRUCacheTest.BasicPutGet
8: [       OK ] LRUCacheTest.BasicPutGet (0 ms)
8: [ RUN      ] LRUCacheTest.Eviction
8: [       OK ] LRUCacheTest.Eviction (0 ms)
8: [ RUN      ] LRUCacheTest.UsageOrderCheck
8: [       OK ] LRUCacheTest.UsageOrderCheck (0 ms)
8: [ RUN      ] LRUCacheTest.ExpirationTime_NoExpirationIfZero
8: [       OK ] LRUCacheTest.ExpirationTime_NoExpirationIfZero (52 ms)
8: [ RUN      ] LRUCacheTest.ExpirationTime_BasicExpiry
8: [       OK ] LRUCacheTest.ExpirationTime_BasicExpiry (65 ms)
8: [ RUN      ] LRUCacheTest.ExpirationTime_ResetOnAccess
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp:129: Failure
8: Expected: switch (0) case 0: default: if (const ::testing::AssertionResult gtest_ar = (::testing::internal::EqHelper::Compare("cache.get(\"K\")", "123", cache.get("K"), 123))) ; else ::testing::internal::AssertHelper(::testing::TestPartResult::kNonFatalFailure, "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/test.cpp", 129, gtest_ar.failure_message()) = ::testing::Message() doesn't throw an exception.
8:   Actual: it throws std::runtime_error with description "Key has expired".
8: 
8: [  FAILED  ] LRUCacheTest.ExpirationTime_ResetOnAccess (220 ms)
8: [ RUN      ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate
8: [       OK ] LRUCacheTest.ExpirationTime_ExistingKeysNotAffectedUntilUpdate (64 ms)
8: [ RUN      ] LRUCacheTest.PutAll_Basic
8: [       OK ] LRUCacheTest.PutAll_Basic (0 ms)
8: [ RUN      ] LRUCacheTest.PutAll_CapacityOverflow
8: [       OK ] LRUCacheTest.PutAll_CapacityOverflow (0 ms)
8: [ RUN      ] LRUCacheTest.PutAll_WithExpirationTime
8: [       OK ] LRUCacheTest.PutAll_WithExpirationTime (65 ms)
8: [----------] 12 tests from LRUCacheTest (468 ms total)
8: 
8: [----------] Global test environment tear-down
8: [==========] 12 tests from 1 test suite ran. (468 ms total)
8: [  PASSED  ] 11 tests.
8: [  FAILED  ] 1 test, listed below:
8: [  FAILED  ] LRUCacheTest.ExpirationTime_ResetOnAccess
8: 
8:  1 FAILED TEST
 8/11 Test  #8: model6_tests .....................***Failed    0.79 sec
test 9
      Start  9: model7_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
UsersUnable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests

9: Test command: 
9: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
 9/11 Test  #9: model7_tests .....................***Not Run   0.00 sec
test 10
      Start 10: model8_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests

10: Test command: 
10: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
10/11 Test #10: model8_tests .....................***Not Run   0.00 sec
test 11
      Start 11: model9_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/mUnable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Errors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
odel9_tests

11: Test command: 
11: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
11/11 Test #11: model9_tests .....................***Not Run   0.00 sec

0% tests passed, 11 tests failed out of 11

Total Test time (real) =   4.12 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Failed)
	  2 - model1_tests (Failed)
	  3 - model10_tests (Not Run)
	  4 - model2_tests (SEGFAULT)
	  5 - model3_tests (Failed)
	  6 - model4_tests (Not Run)
	  7 - model5_tests (SEGFAULT)
	  8 - model6_tests (Failed)
	  9 - model7_tests (Not Run)
	 10 - model8_tests (Not Run)
	 11 - model9_tests (Not Run)
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build
      Start  1: incorrect_solution_tests
 1/11 Test  #1: incorrect_solution_tests .........***Failed    0.48 sec
      Start  2: model1_tests
 2/11 Test  #2: model1_tests .....................***Failed    0.48 sec
      Start  3: model10_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model10_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model10_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model10_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model10_tests
 3/11 Test  #3: model10_tests ....................***Not Run   0.00 sec
      Start  4: model2_tests
 4/11 Test  #4: model2_tests .....................***Exception: SegFault  0.15 sec
      Start  5: model3_tests
 5/11 Test  #5: model3_tests .....................***Failed    0.47 sec
      Start  6: model4_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model4_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model4_tests
 6/11 Test  #6: model4_tests .....................***Not Run   0.00 sec
      Start  7: model5_tests
 7/11 Test  #7: model5_tests .....................***Exception: SegFault  0.14 sec
      Start  8: model6_tests
 8/11 Test  #8: model6_tests .....................***Failed    0.47 sec
      Start  9: model7_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model7_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model7_tests
 9/11 Test  #9: model7_tests .....................***Not Run   0.00 sec
      Start 10: model8_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model8_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model8_tests
10/11 Test #10: model8_tests .....................***Not Run   0.00 sec
      Start 11: model9_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model9_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Release/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Debug/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/MinSizeRel/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/RelWithDebInfo/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Deployment/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model9_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Development/model9_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/model9_tests
11/11 Test #11: model9_tests .....................***Not Run   0.00 sec

0% tests passed, 11 tests failed out of 11

Total Test time (real) =   2.21 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Failed)
	  2 - model1_tests (Failed)
	  3 - model10_tests (Not Run)
	  4 - model2_tests (SEGFAULT)
	  5 - model3_tests (Failed)
	  6 - model4_tests (Not Run)
	  7 - model5_tests (SEGFAULT)
	  8 - model6_tests (Failed)
	  9 - model7_tests (Not Run)
	 10 - model8_tests (Not Run)
	 11 - model9_tests (Not Run)
Errors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621208/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
